dictionary.go

```diff
@@ -349,7 +349,10 @@ func (page *indexedPage) RepetitionLevels() []int8 { return nil }
 
 func (page *indexedPage) DefinitionLevels() []int8 { return nil }
 
-func (page *indexedPage) WriteTo(e encoding.Encoder) error { return e.EncodeInt32(page.values) }
+func (page *indexedPage) WriteTo(e encoding.Encoder) error {
+	err := e.EncodeInt32(page.values)
+	return err
```

Comment:
Was this done by a formatting tool?

Reply:
No, another troubleshooting artifact (I just wanted to print the error)